_about/CONTRIBUTING.md

```diff
@@ -57,7 +57,7 @@ To aid local development, we can use [development overrides for Terraform provid
 # ~/.terraformrc
 provider_installation {
   dev_overrides {
-    "prefecthq/prefect" = "/Users/johnsmith/code/terraform-provider-prefect/build"
+    "hashicorp/prefect" = "/Users/johnsmith/code/terraform-provider-prefect/build"
```

Comment:
im not quite sure what changed here, but the default provider path kept looking for `hashicorp/prefect`